### PR TITLE
BLKN: Build & Deploy settings

### PR DESCRIPTION
**Pull request by Fallion**
- Adds Babel export scripts
- Adds prepublish script
- Adds Flow file export script
- Adds npmrc scope settings

### CODE REVIEW

File: blkn/package.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
   "name": "@kiwicom/balkan-ui",
-  "version": "0.0.0-rc0",
+  "version": "0.0.0-rc2",
   "scripts": {
-    "storybook": "start-storybook -p 6007 -c .storybook"
+    "storybook": "start-storybook -p 6007 -c .storybook",
+    "build": "babel --out-dir lib --ignore *.stories.js,*.test.js src",
```
Comment (oreqizer) [inline blkn/package.json]: how about building also an `es` version? see the Kiwi part for an example

Comment (Fallion) [reply blkn/package.json]: Updated. Please review again.

File: blkn/package.json [hunk 2]
```diff
@@ -5,11 +5,14 @@
     "storybook": "start-storybook -p 6007 -c .storybook",
     "build": "babel --out-dir lib --ignore *.stories.js,*.test.js src",
     "build:flow": "flow-copy-source -v -i '**/*.stories.js,**/*.test.js' src lib",
-    "clean": "rimraf lib",
-    "prepare": "npm run clean && npm run build && npm run build:flow"
+    "build:module":
+      "babel --presets react --no-babelrc --out-dir es --ignore *.stories.js,*.test.js src ",
+    "clean": "rimraf lib && rimraf es",
```
Comment (oreqizer) [inline blkn/package.json]: `rimraf lib es`

Comment (Fallion) [reply blkn/package.json]: Done

**Pull request merged**